superset-frontend/src/explore/actions/saveModalActions.js

```diff
@@ -215,3 +218,23 @@ export const getDashboard = dashboardId => async dispatch => {
     throw error;
   }
 };
+
+//  Get dashboards the slice is added to
+export const getSliceDashboards = slice => async dispatch => {
+  if (slice) {
+    try {
+      const response = await SupersetClient.get({
+        endpoint: `/api/v1/chart/${slice.slice_id}?q=${rison.encode({
```

Comment:
nits:
```suggestion
        endpoint: `/api/v1/chart/${slice?.slice_id}?q=${rison.encode({
```

Reply:
I check above that `slice` is defined, so I don't think this check is necessary.